exercises/pythagorean-triplet/tests/pythagorean-triplet.rs

```diff
@@ -1,6 +1,79 @@
 extern crate pythagorean_triplet;
 
+use pythagorean_triplet::find;
+
+fn process_tripletswithsum_case(sum: u32, expected: Vec<[u32; 3]>) {
+    if !expected.is_empty() {
+        let triplets = find(sum);
+
+        assert!(
+            expected
+                .iter()
+                .all(|expected_triplet| triplets.contains(expected_triplet))
```

Comment:
This line IS necessary, but I think another check is necessary. Otherwise, an impl that adds `[0, 0, 0]` to any non-empty vec will also pass the tests, since it doesn't check that every element of `triplets` should be there.

Try some of the ways used before. Perhaps check the `len` as in https://github.com/exercism/rust/blob/master/exercises/allergies/tests/allergies.rs#L5-L21 . Or use a HashSet as in https://github.com/exercism/rust/blob/master/exercises/anagram/tests/anagram.rs#L6-L12

Reply:
or sorting the input, as in https://github.com/exercism/rust/blob/master/exercises/saddle-points/tests/saddle-points.rs#L6-L10

There appear to be many exercises where we use an ordered data structure but don't actually care about the order, and we solved it in a different way each time, that is quite interesting.

Reply:
If the `HashSet` approach is used, should the stub function also return `HashSet<String>`?

Reply:
It must, otherwise it would not compile